mdoc/src/main/scala-3/mdoc/MainSettings.scala

```diff
@@ -0,0 +1,74 @@
+package mdoc
```

Comment:
This is copied and stripped down version of the Scala 2 `MainSettings` with only methods used by `Mdoc.java` interface.